admin-notice/class-admin-notice-controller.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Automattic\VIP\Admin_Notice;
+
+// function sample_admin_notice__error() {
+//     $class = 'notice notice-info';
+//     $message = __( 'Irks! An error has occurred.', 'sample-text-domain' );
+
+//     printf( '<div class="%1$s"><p>%2$s</p></div>', esc_attr( $class ), esc_html( $message ) );
+// }
+// add_action( 'admin_notices', 'sample_admin_notice__error' );
```

Comment:
If this is meant to be an example of how to use it, then either put it as a proper DocBlock, or move it to a readme.

Reply:
Nah, sorry about that. That was just for me to learn how to do it and I forgot to remove it. Removed now.